src/embed-receipt.ts

```diff
@@ -182,17 +173,20 @@ export class RWPEmbedReceipt extends LitElement {
   }
 
   render() {
-    let {
-      numValid,
-      numInvalid,
+    assertNonNullish(this.url, "url");
+    assertNonNullish(this.ts, "ts");
+    assertNonNullish(this.collInfo, "collInfo");
+    assertNonNullish(this.appLogo, "appLogo");
```

Comment:
Perhaps we can handle nullish checks in `html`, so that the app will continue to render even when noncritical data like `appLogo` is missing. Maybe this can be pulled into a separate PR?

Reply:
Sure, that works for me! I presume you mean skipping rendering things that depend on these if they're not present?